demo-app/src/main/java/com/gravatar/demoapp/ui/DemoGravatarApp.kt

```diff
@@ -147,7 +147,7 @@ private fun GravatarTabs(
 
 @Composable
 private fun ProfileTab(modifier: Modifier = Modifier, onError: (String?, Throwable?) -> Unit) {
-    var email by remember { mutableStateOf("gravatar@automattic.com", neverEqualPolicy()) }
+    var email by remember { mutableStateOf("maxime.biais@automattic.com", neverEqualPolicy()) }
```

Comment:
This helps a lot with the default card, but I think we should create a set of live test accounts and eventually have some kind of autofill in the email text field for quickly switching profiles

Reply:
I was talking with @mlumeau before and we are not sure if we should harcode private mails in the code. Wdyt?

Reply:
This address is already crawlable from some public places (commits, pom files), so it's not a big deal. Would be better to have live test accounts though, I can create one before merging this.